pkg/client/leaderelection/leaderelection.go

```diff
@@ -296,3 +308,50 @@ func (le *LeaderElector) tryAcquireOrRenew() bool {
 	le.observedTime = time.Now()
 	return true
 }
+
+func DefaultLeaderElectionCLIConfig() LeaderElectionCLIConfig {
+	return LeaderElectionCLIConfig{
+		LeaderElect:   false,
+		LeaseDuration: 15 * time.Second,
```

Comment:
given that these are consts... 
could we place the consts at the top, at least for consistency with other libs as they are cleaned.